### PR TITLE
feat: support to change window effects

### PR DESCRIPTION
**Pull request by xionglinlin**
add two interface setWindowEffect and setWindowStartUpEffect

Issue: https://github.com/linuxdeepin/developer-center/issues/10322

### CODE REVIEW

File: qmlplugin/qmlplugin_plugin.cpp [hunk 1]
```diff
@@ -206,6 +207,7 @@ void QmlpluginPlugin::registerTypes(const char *uri)
     dtkRegisterUncreatableType<DColor>(uri, implUri, 1, 0, "Color",
                                        QStringLiteral("Color is only available as enums."));
     dtkRegisterUncreatableType<DPopupWindowHandle>(uri, implUri, 1, 0, "PopupHandle", "PopupWindow Attached");
+    dtkRegisterUncreatableType<DPlatformHandle>(uri, implUri, 1, 0, "PlatformHandle", "PlatformHandle Attached");
```
Comment (18202781743) [inline qmlplugin/qmlplugin_plugin.cpp:210]: 不是attached属性,

Comment (xionglinlin) [reply qmlplugin/qmlplugin_plugin.cpp]: done

**Review by 18202781743 — APPROVED** [review]

**Pull request merged**